stable/redis/ci/production-values.yaml

```diff
@@ -353,3 +353,16 @@ volumePermissions:
 ##
 configmap: |-
   # maxmemory-policy volatile-lru
+
+## Sysctl InitContainer
+## used to perform sysctl operation to modify Kernel settings (needed sometimes to avoid warnings)
+sysctlImage:
+  enabled: true
```

Comment:
This is going to modify the Kernel settings of the cluster used in the CI... I think we should disable it.

Reply:
@cpanato asked to add ci values, I now replaced the command with a dummy echo.

Reply:
👍 great